src/dlgTriggerEditor.cpp

```diff
@@ -664,6 +664,13 @@ void dlgTriggerEditor::slot_item_selected_search_list(QTreeWidgetItem* pItem, in
                 slot_trigger_selected( treeWidgetItem );
                 treeWidget_triggers->setCurrentItem( treeWidgetItem, 0 );
                 treeWidget_triggers->scrollToItem( treeWidgetItem );
+
+                // The next line is on Vadi's direct and adamant orders. I suspect substance will
+                // hit ventilators. This code will select all found instances of the search term,
+                // and create a caret for each so all instance will be edited by the user at the
+                // same time.
```

Comment:
_Well, that __might__ be of use sometimes_!  But it ought not to be hard-coded unless the extra carets go away if the user actually selects one of them _before_ typing any replacement text.

And what if a caret is put on something that cannot be edited (like functions names in the "Variables" tree)?